data/l10n/bzflag_sv.po

```diff
@@ -1276,10 +1276,10 @@ msgstr "högt uppe den är: Ju större prick, desto högre upp. Flaggor är små
 #msgid "Toggle Main Flags:"
 #msgstr ""
 
-#msgid "Silience/UnSilence"
+#msgid "Silence/Unsilence"
```

Comment:
There are more instances of this typo in the other language files. The src/bzflag/HelpMenu.cxx and src/bzflag/KeyboardMapMenu.cxx have the S in UnSilence capitalized. That's unnecessary there as well. The data/l10n/bzflag_lt.po has a typo on the line for Silence/UnSilence that reads "# sgid" instead of "#msgid".